ui/app/AppLayouts/Wallet/views/TransactionDetailView.qml

```diff
@@ -68,6 +77,21 @@ Item {
         }
     }
 
+    Connections {
+        target: RootStore.history
+        function onTxDecoded(txHash: string, dataDecoded: string) {
+            if (!root.isTransactionValid || txHash !== root.transaction.txHash || !dataDecoded)
+                return
+            const decodedObject = JSON.parse(dataDecoded)
```

Comment:
This can potentially throw an exception

Reply:
On second thought dataDecoded can be true and still not parse properly. 

Added the try catch